hushline/model.py

```diff
@@ -31,9 +31,23 @@ def default(cls) -> "SMTPEncryption":
 class HostOrganization(Model):
     __tablename__ = "host_organization"
 
-    id: Mapped[int] = mapped_column(primary_key=True, default=1)
-    brand_app_name: Mapped[str] = mapped_column(db.String(255), default="🤫 Hush Line")
-    brand_primary_hex_color: Mapped[str] = mapped_column(db.String(7), default="#7d25c1")
+    _DEFAULT_PRIMARY_KEY: int = 1
+    _DEFAULT_BRAND_PRIMARY_HEX_COLOR: str = "#7d25c1"
+    _DEFAULT_BRAND_APP_NAME: str = "🤫 Hush Line"
+
+    id: Mapped[int] = mapped_column(primary_key=True)
+    brand_app_name: Mapped[str] = mapped_column(db.String(255), default=_DEFAULT_BRAND_APP_NAME)
+    brand_primary_hex_color: Mapped[str] = mapped_column(
+        db.String(7), default=_DEFAULT_BRAND_PRIMARY_HEX_COLOR
+    )
+
+    @classmethod
+    def default(cls) -> Self | None:
+        return db.session.get(cls, cls._DEFAULT_PRIMARY_KEY)
+
+    def __init__(self, primary_key: int | None = None) -> None:
```

Comment:
What's the purpose of manipulating the primary key in this way?

Reply:
The overall intent is to be sure on record creation of the primary key value which is used for created records. I see depending on the implementation detail of how the database automatically assigns primary keys as a coupling which is unnecessary. The [review](https://github.com/scidsg/hushline/pull/573#discussion_r1764484915) by @brassy-endomorph also requested that the primary key not directly be given a default value in the ``mapped_column`` function. So, this initializer is intended meet that functionality while adhering to the request.

Reply:
I don't see where the parameter is provided, should it be specified here, https://github.com/scidsg/hushline/pull/573/files#diff-567f5ecf57aaf3736dae09a470c91753792589003ee6b4ec067651aee9669afbR83

Reply:
Ah, my bad, I was misreading this, I'm fine with this.